compiler/minmax-embedder/requires.cmake

```
@@ -0,0 +1 @@
+require("loco")
```

Is it requires "loco" only?

```suggestion
require("safemain")
require("loco")
require("luci")
require("arser")
require("vconone")
```

---

`arser`, `vconone`, `safemain` is not required. But `luci` is required. I will add it.

---

Then why `arser`, `vconone`, and `safemain` is linked to `minmax_embedder` in cmake's `target_link_libraries` ?

---

@hseok-oh You're right. I removed them from library dependencies. They should be linked for `driver`, which will be the next PR.